### PR TITLE
Add documentation for Tiered Tooltips

### PR DESCRIPTION
**Pull request by OscarXcore**

### CODE REVIEW

**Comment by OscarXcore** [comment]
I am not sure if the docs for this should be put under GameStages or not. Also its been roughly 10 days since I submitted this, and I have no clue why it hasnt been merged or if there is anything wrong. (My bad if my tone comes off wrong, I have been working nights and its a toll on me)

**Comment by OscarXcore** [comment]
Moved it under GameStages/Other/Tiered Tooltips

File: mkdocs.yml [hunk 1]
```diff
@@ -252,6 +252,8 @@ pages:
       - RecipeStages: 'Mods/GameStages/RecipeStages/RecipeStages.md'
       - TinkerStages: 'Mods/GameStages/TinkerStages/TinkerStages.md'
       - WailaStages: 'Mods/GameStages/WailaStages/WailaStages.md'
+      - Other:
```
Comment (jaredlll08) [inline mkdocs.yml]: It should be in the normal GameStages folder, no need for sub folders.

**Comment by OscarXcore** [comment]
All fixed up.

**Pull request merged**